lti_consumer/plugin/views.py

```diff
@@ -241,8 +246,8 @@ def deep_linking_response_endpoint(request, lti_config_id=None):
             },
             status=403
         )
-    except KeyError:
-        log.info("LTI Content Type not supported.")
+    except KeyError as exc:
```

Comment:
This exception type is too generic for so large a try block.  How can we be sure that KeyErrors mean the content type is not supported?  Can we move this error handling to a narrower chunk of code?  And also include the content type in the message.

Reply:
I used a specific exception for this case since there was already one in place: `LtiDeepLinkingContentTypeNotSupported`.